tools/packaging/packaging.psm1

```diff
@@ -2971,3 +2971,84 @@ function Get-PackageVersionAsMajorMinorBuildRevision
 
     $packageVersion
 }
+
+function New-DotnetSdkContainerFxdPackage {
+    param(
+        [Parameter(Mandatory)] $FxdPackagePath,
+        [Parameter(Mandatory)] $ReleaseTag
+    )
+
+    $Version = $ReleaseTag -Replace '^v'
+
+    if ($IsWindows) {
+        $basePackagePattern = "*$Version-win-fxdependent.zip"
+        $packageNamePlatform = 'win'
+        $packageNameExtension = '.zip'
+    } else {
+        $basePackagePattern = "*$Version-linux-x64-fxdependent.tar.gz"
+        $packageNamePlatform = 'linux-x64'
+        $packageNameExtension = '.tar.gz'
+    }
+
+    $packageName = "powershell-$Version-$packageNamePlatform-fxd-dotnetsdk$packageNameExtension"
+
+    ## Get fxdependent package path
+    $fxdPackage = Get-ChildItem $FxdPackagePath -Recurse -Filter $basePackagePattern
+
+    Write-Verbose -Verbose "Fxd Package Path: $fxdPackage"
+
+    if ($fxdPackage) {
+        ## Extract fxd package
+        $tempExtractFolder = New-Item -Type Directory -Path "$FxdPackagePath/fxdreduced" -Force
+        Push-Location $tempExtractFolder
+
+        Write-Verbose -Verbose "Pushed location"
```

Comment:
Seems like this would be more useful if the pushed location is included in the message.

Reply:
Added. Thanks!